losses/multiclass_losses.py

```diff
@@ -136,7 +137,7 @@ def loss(y_true: tf.Tensor, y_pred: tf.Tensor) -> tf.Tensor:
         y_pred = K.clip(y_pred, K.epsilon(), 1-K.epsilon())  # To avoid unwanted behaviour in K.log(y_pred)
 
         # p * log(p̂) * class_weights
-        wce_loss = y_true * K.log(y_pred) * class_weights
+        wce_loss = y_true * K.log(y_pred + epsilon) * class_weights
```

Comment:
Same here, to what I can see, this should be handled by the clipping in the line above? Or?